transmart-rest-api/grails-app/controllers/org/transmartproject/interceptors/SystemAuditInterceptor.groovy

```diff
@@ -0,0 +1,19 @@
+package org.transmartproject.interceptors
+
+import groovy.transform.CompileStatic
+
+@CompileStatic
+class SystemAuditInterceptor extends AuditInterceptor {
+
+    SystemAuditInterceptor() {
+        match(controller: ~/system/, action: ~/afterDataLoadingUpdate/)
+    }
+
+    boolean after() {
+        def ip = getIP()
```

Comment:
This line is not needed, right? Groovy will try to call `getIp()` if you use `${ip}`.